### DISCUSSION TITLE
style in a table

### DISCUSSION THREAD

**Discussion by flosigime** [sqlpage/SQLPage#264] [Q&A]
Hello all, 

I would like to know how i can use class or id in a table. 
I have this : 
```sql
select 
'table' as component,
'name and adresses' as title;

select
a.name as Nom,
a.adress as Adresse,
a.city as Ville
from adresses as a;

```
is it possible to use a specific class or id on a column, 'ville' for example ? 
Thank you for the answer...

**Comment by lovasoa** [comment]
In html, tables are represented as cells (`<td>`) nested in rows (`<tr>`)  . So there isn't a column element for you to assign an id to. What are you trying to achieve?

**Comment by flosigime** [comment]
Hello,
I'm looking for a solution to hide some columns in a table on small screen.
On an old website (and first try to create a crud web app) i used a css
class on some <th> and <td> cells in my table. After an @media (max-width:
800px), the cells with the specific class were not displayed when screen
was under 800 px..

**Comment by lovasoa** [comment]
I see!

Each cell already has a class with the name of the column. However, unfortunately, at the moment, the class is not applied to the header. I'll change that.

So, at the moment, you can use

```css
.Ville, th:nth-child(3) {
  display: none;
}
```

and it the next version, you'll be able to do simply

```css
.Ville {
  display: none;
}
```

**Comment by flosigime** [comment]
Hello,
Thank you for the tip.
And if i want to hide the second column, i write  : 

`
.Adresse, th:nth-child(2) { 
display: none; 
}
`
